ui/pages/add-collectible/add-collectible.js

```diff
@@ -46,8 +50,71 @@ export default function AddCollectible() {
   );
   const [tokenId, setTokenId] = useState('');
   const [disabled, setDisabled] = useState(true);
+  const [eventTracked, setEventTracked] = useState([
+    {
+      event: 'Token Added',
+      address: '',
+      tracked: false,
+    },
+  ]);
+
   const [collectibleAddFailed, setCollectibleAddFailed] = useState(false);
 
+  const trackEvent = useContext(NewMetaMetricsContext);
+  const collectibles = useSelector(getCollectibles);
+
+  useEffect(() => {
+    if (!tokenId || !address) {
+      return;
+    }
+
+    const newCollectible = collectibles.find(
+      (collectible) =>
+        collectible.address === address &&
+        collectible.tokenId === tokenId.toString(),
+    );
+
+    if (!newCollectible) {
+      return;
+    }
+
+    const alreadyTracked = eventTracked.find(
+      (et) => et.address === address && et.tracked === true,
+    );
+
+    if (alreadyTracked) {
+      return;
+    }
+
+    trackEvent({
+      event: 'Token Added',
+      category: 'Wallet',
+      sensitiveProperties: {
+        token_symbol: '',
```

Comment:
best to leave these empty strings as undefined (just omit the properties), segment will strip them anyways.

Reply:
Fixed [here](https://github.com/MetaMask/metamask-extension/pull/14279/commits/1fd076232dab0526122622ef0968e9f664518c6d)

Reply:
You could add this field back now since some NFTs have a symbol and it is returned by the `getTokenStandardAndDetails` method that you are using now.

Reply:
cc @segun ☝️

Reply:
Somehow missed this comment....fixed [here](https://github.com/MetaMask/metamask-extension/pull/14279/commits/b2b69c05d13ded9b2180dcda2e1afe06c9a004bf)

Reply:
Somehow missed that....fixed [here](https://github.com/MetaMask/metamask-extension/pull/14279/commits/b2b69c05d13ded9b2180dcda2e1afe06c9a004bf)

Reply:
@segun that commit doesn't see to re-add the `token_symbol` to the event?